procfs/procfs.go

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright 2015 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package procfs
+
+// ProcInterface that presents GetFullContainerName API implemented
+// by ProcFS and FakeProcFS
+type ProcInterface interface {
```

Comment:
Maybe just `Interface` then if golint is complaining?

Reply:
golint is not complaining about this one. It was complaining about ProcFSInterface. I just thought that ProcInterface is closer to the original name.